### DISCUSSION TITLE
付了1年的费用，但使用的付款邮箱一直无法收到激活码

### DISCUSSION THREAD

**Discussion by johnniev5** [getlantern/lantern-forum-cn#647] [General]
如题，付款邮箱无法正常收到激活码，6月17号付的款，已经好几天了，具体不知道什么原因，是不是链接地址被劫持了？我的付款邮箱为，请尽快帮忙解决，多谢！如果无法解决，就需要申请退款了，谢谢！

@Mike777Khoo 这个链接https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ，我也已经提交了，请尽快帮忙处理，多谢了！

**Comment by MaxMKMax** [comment]
@johnniev5 您好. 请放心. 我们的客服会处理好. 请给我们两天时间. 这是因为国内限制了邮件的发送, 所以您没收到邮件激活码了. 既然您已经提交了, 我就帮您删除掉您的个人资料.

**Comment by MaxMKMax** [comment]
@johnniev5 删除好了. 我也另外帮您提醒相关部门了.
